src/domain/entities/figma-design-identity.test.ts

```diff
@@ -0,0 +1,139 @@
+import { FigmaDesignIdentity } from './figma-design-identity';
+import {
+	MOCK_FIGMA_FILE_IDENTITY,
+	MOCK_FIGMA_FILE_KEY,
+	MOCK_FIGMA_NODE_ID,
+	MOCK_FIGMA_NODE_IDENTITY,
+} from './testing';
+
+describe('FigmaDesignIdentity', () => {
+	describe('constructor', () => {
+		it('should throw when fileKey is empty string', () => {
+			expect(() => new FigmaDesignIdentity('')).toThrow();
+		});
+
+		it('should throw when nodeId is empty string', () => {
+			expect(() => new FigmaDesignIdentity(MOCK_FIGMA_FILE_KEY, '')).toThrow();
+		});
+	});
+
+	describe('fromFigmaDesignUrl', () => {
+		it('should return identity when Atlassian design ID does not contain node part', () => {
+			const result = FigmaDesignIdentity.fromAtlassianDesignId(
+				`${MOCK_FIGMA_FILE_KEY}/${MOCK_FIGMA_NODE_ID}`,
```

Comment:
I think the Atlassian design ID isn't supposed to have a node ID for this test.

Reply:
Oh, great catch! I copied/pasted this test too many times and finally pasted an incorrect one 🙈